### PR TITLE
Report duplicate entities left by the 2026.7 unique-ID migration as a fixable Repair

### PR DESCRIPTION
**Pull request by Optic00**
The 2026.7 unique-ID migration keeps the older working registry entry when an entity exists twice, because dashboards, automations, history, and long-term statistics point at it. The model-named duplicate keeps its legacy unique ID and stays unavailable; until now it only surfaced as a log warning most users never see (#649).

This PR turns that leftover into a persistent, fixable Repair per config entry:

- The Repair lists the exact duplicate entity IDs and is created, updated, or deleted on every migration run.
- The fix flow shows the IDs again, tells the user to check dashboard and automation references first, and states that long-term statistics are neither merged nor deleted; removing those stays a manual step in Developer Tools.
- The duplicate list is recomputed immediately before removal, so a stale Repair cannot delete entries that no longer collide.
- Removal happens only after explicit confirmation; automatic cleanup is deliberately out because deleting registry entries is destructive.

RepairsFlowManager passes its init data as `user_input` to `async_step_init`. A regression test on that real path proves that opening the Repair never counts as confirmation; the confirm form is always shown.

Verified with `ruff format --check`, `ruff check`, `mypy`, and the test suite (896 passed, 1 skipped).

## Summary by Sourcery

Introduce a fixable, persistent Repair for duplicate entities left by the unique-ID migration and ensure related issues are safely created, synchronized, and cleaned up.

New Features:
- Add a Repairs-based flow to list and optionally remove duplicate entity registry entries per config entry.
- Expose a dedicated duplicate-entity Repair issue type tied to each config entry and controller model.

Bug Fixes:
- Ensure removing a config entry clears both controller and duplicate-entity Repair issues and that malformed Repair data cannot trigger entity deletion.
- Prevent stale Repairs from deleting entities that are no longer in a duplicate state by revalidating losers before removal.

Enhancements:
- Replace the previous transient log-only duplicate warning with clearer guidance and safer wording about long-term statistics handling.
- Document the preferred cleanup path for unavailable duplicate entities and clarify how statistics are preserved and not auto-merged.
- Refine translation strings and tests so duplicate-entity Repairs use safe placeholders and consistent messaging across languages.

Tests:
- Add extensive tests for the new duplicate-entity Repair flow, including confirmation behavior, stale and malformed cases, and interaction with the Repairs flow manager.
- Extend migration, init, and translation tests to cover duplicate-entity Repairs, updated logging text, and issue lifecycle behavior.

### CODE REVIEW

File: tests/test_migration.py [hunk 1]
```diff
@@ -247,6 +251,24 @@ async def test_configured_name_wins_over_the_replacement_entity(
...
+    issue = ir.async_get(hass).async_get_issue(
+        DOMAIN, f"duplicate_entities_{config_entry_with_name.entry_id}"
+    )
+    assert issue is not None
+    assert issue.is_fixable is True
+    assert issue.is_persistent is True
+    assert issue.severity is ir.IssueSeverity.WARNING
+    assert issue.data == {
+        "entry_id": config_entry_with_name.entry_id,
+        "model_id": ControllerModel.WPM_3.value,
```
[the remark this answers is not in the document]
Comment (Optic00) [reply tests/test_migration.py:266]: Correction: #650 had already been merged before this reply. Commit a96b1b1 exists only on my fork branch and is not part of the merged PR. The assertion does catch a missing issue_domain, but getting it upstream would require a separate follow-up PR.

File: custom_components/stiebel_eltron_isg/repairs.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+        return self.async_show_form(
+            step_id="confirm",
+            description_placeholders={
+                "count": str(len(entity_ids)),
+                "entities": "\n".join(f"- `{entity_id}`" for entity_id in entity_ids),
+            },
+        )
+
+
+async def async_create_fix_flow(
```
[the remark this answers is not in the document]
Comment (Optic00) [reply custom_components/stiebel_eltron_isg/repairs.py:73]: I kept the current guard-clause implementation. The function has one short validation sequence and returns early for malformed data. Extracting a NamedTuple, a parser helper, and a separate integer helper would add more surface without changing behavior. There is no duplicate validation to consolidate, and the malformed-data tests already cover missing data, wrong types, missing entries, invalid models, and mismatched issue IDs.
